spec/que/job_spec.rb

```diff
@@ -83,11 +89,13 @@ def expected_job_count
           enqueue_method.call({'a' => 1, 'b' => 2}, c: 3, d: 4)
           execute
           assert_equal([{a: 1, b: 2}], $args)
+          assert_equal({:c=>3, :d=>4}, $kwargs)
```

Comment:
Any reason to use hash rockets rather than colons here?

Reply:
Ah, we don't have RuboCop yet. Yeah, let's use colon syntax for consistency